examples/verifyIdToken.js

```diff
@@ -32,7 +32,7 @@ async function main() {
     // Verify the id_token, and access the claims.
     const ticket = await oAuth2Client.verifyIdToken({
       idToken: oAuth2Client.credentials.id_token,
-      audience: keys.web.client_id,
+      audience: keys.web.client_id
```

Comment:
Did you manually make this change or was this clang-format? Seems like an unnecessary change, but perhaps the version of clang-format has changed?

Reply:
This is prettier.  I don't have prettier linting blocking npm test right now.  I will go fix that!